frontend/src/app/main/ui/workspace/tokens/modals/import.cljs

```diff
@@ -152,11 +169,15 @@
                                          (= type "application/json")
                                          (str/ends-with? name ".json"))))))]
              (->> (rx/from files)
-                  (rx/mapcat (fn [file]
-                               (->> (wapi/read-file-as-text file)
-                                    (rx/map (fn [file-text]
-                                              [(.-webkitRelativePath file)
-                                               file-text])))))
+                  (rx/map (fn [file]
+                            (->> (wapi/read-file-as-text file)
+                                 (rx/map (fn [file-text]
+                                           [(.-webkitRelativePath file)
+                                            file-text])))))
+                  (rx/mapcat (fn [entries]
+                               (prn "entries" entries)
+                               entries))
+                  (validate-token-files)
```

Comment:
I think this code should be reverted (i guess the main intention of this change is put that `prn`)